ports/poolstl/portfile.cmake

```
@@ -0,0 +1,19 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO alugowski/poolSTL
+    REF "v${VERSION}"
+    SHA512 a2d29056b29f32f034560f88e05f7257ff1f43b46579b940da3e340c97cf8bfbb7d886f5101044d5e22931af37bbcf72f956a0157e135cdf10c7a987e56ba081
+    HEAD_REF main
+    PATCHES fix-find-dependency.patch
```

If you already changing the port:
```suggestion
    PATCHES
        fix-find-dependency.patch
```

---

Applied in 3702fd30b371fa0c771fc7ced134f24731ca4b9f.